docs/wiki/feature/atragmx.md

```diff
@@ -14,19 +14,63 @@ version:
 
 ## Overview
 
-### Sub-feature 1
-Short description of sub-feature 1.
+### AtragMx from HORUS VISION
 
-### Sub-feature 2
-Short description of sub-feature 2.
+Horus ATragMX software considers atmospheric conditions, gun data, ammunition, range, speed and muzzle velocity to calculate precise aiming solutions with "come-up" results – and even accounts for Coriolis and spin drift effects. ATragMX, loaded on a handheld computer made by TDS Recon, is easy to use and lightning-fast. The Recon meets the rigorous MIL-STD-810F military standard for drops, vibration, humidity, altitude and extreme temperatures.
 
+### Official Manual and Horus Videos
```

Comment:
Move this section to the end, just above Dependencies. And change to `### 3. <text>`